lib/fiware-orion-pep.js

```diff
@@ -126,6 +126,21 @@ function createDynamicMiddlewareExecutor(proxyObj) {
     };
 }
 
+function traceRequest(req, res, next) {
+    logger.debug('Request for path [%s] from [%s]', req.path, req.get('host'));
+    logger.debug('Headers:\n%j\n', req.headers);
+
+    if (req.headers['content-type'] === 'application/json') {
+        logger.debug('Body:\n\n%s\n\n', JSON.stringify(req.body, null, 4));
```

Comment:
Not exactly the same case, as this middleware isn't even loaded when the log level is not DEBUG... I'm not sure between changing it for coherence or leaving it as it is for ease of debugging.